wg-lts/charter.md

```diff
@@ -0,0 +1,139 @@
+components to run on cluster nodes. This brings in additional dimensions
+of version skew and does so with more external components and support
+matrix complexity.  These are developers in, for example:
+* SIG Storage (CSI)
+* SIG Network (CNI)
+* SIG Node (CRI)
+* SIG Cloud Provider
+  * SIG AWS
+  * SIG Azure
+  * SIG GCP
+  * SIG IBMCloud
+  * SIG OpenStack
+  * SIG VMware
+* SIG Cluster Lifecycle
+
+There is yet another set of developers of Kubernetes internals who are
+those involved in meta-topics:
+* SIG Release: production of supported release artifacts
+* SIG Testing: how we can most effectively test Kubernetes
+* Product Security Team (PST): security vulnerability handling
+* SIG Architecture: maintains and evolves the design principles of Kubernetes, and provides a consistent body of expertise necessary to ensure architectural consistency over time.  Also defines conformance testing.
+* Steering Committee: scope includes deciding how and when official releases of Kubernetes artifacts are made and what they include
+
+Then of course there is also a variety of types of users of Kubernetes:
+* Kubernetes end users
+* Kubernetes cluster operators
+* Kubernetes vendors, distributions, and hosting providers
+These have their own operational complexities and desires around support
+duration and supported upgrade version skews.
+
+## Deliverables
+The specific deliverable is somewhat To-Be-Determined based on
+initial survey and analysis by the WG.
+
+The initial goal is to enumerate the many and often conflicting
+desires around support, version skew, and upgrade path, with an eye
+for possible areas for improvement.  Today these topics are covered
+in a disjoint set of documents across multiple repositories and the
+official Kubernetes website, so even this enumeration is complicated.
+
+On the one hand the WG's surveys might lead to something larger and
+strategic like a community agreed and implementable KEP returned
+to SIG Release to operationalize.  This will come from discussion
+and analysis of the release frequency and support duration relative
+to adoption and adoption blocking issues.
+
+On the other hand it might lead to more tactical project improvements
+that result in a culture of and releases characterized by higher
+quality and stability, such as:
+* Backported patches more rigorously enforced to only contain critical fixes
+* Critical APIs are stable (vN, not vNbetaX)
+* kubernetes/kubernetes repo master branch kept in a releasable state
+* More clear testing signal
+* Higher test coverage
+* Meaningful version-skew, upgrade, and downgrade tests that are immediately fixed, or patches reverted, when broken
+* Reliable API schema upgrades and downgrades
+* Greater supported version skew for kubectl and client-go
+
+## Timeline and Disbanding
+Working Groups are intended to be time limited endeavors.  We expect
+to survey the state of support in late 2018, especially using KubeCon
```

Comment:
This made sense when we pushed it. Survey may not be ready by the KubeCon US.

Reply:
I think "survey" as a general verb is still applicable here.  There is a survey noun in the works as a part of that.